roles/manifest/README.md

```diff
@@ -0,0 +1,69 @@
+theforeman.foreman.manifest
+===========================
+
+Upload Subscription Manifest to Katello
+
+Requirements
+------------
+
+- `theforeman.foreman` collection
+
+Role Variables
+--------------
+
+- `server_url`: URL of the Foreman server.
+- `username`: Username accessing the Foreman server.
+- `password`: Password of the user accessing the Foreman server.
+- `validate_certs`: Whether or not to verify the TLS certificates of the Foreman server.
+- `organization`: Organization to upload Manifest to.
```

Comment:
Should we drop these now that they are global? More just enter a line that refers to the global README? I'm not sure, just trying to think and establish a pattern that reduces overhead and possibility of having outdated docs due to multiple places.

Reply:
I'm thinking we could have a line with a link like:

```
This role supports the common role variables - (link to https://github.com/theforeman/foreman-ansible-modules/pull/978/files#diff-04c6e90faac2675aa89e2176d2eec7d8R87)

Reply:
It's now updated with a link that should become working once the PR is merged.